myosuite/envs/myo/myochallenge/bimanual_v0.py

```diff
@@ -0,0 +1,407 @@
+""" =================================================
+# Copyright (c) Facebook, Inc. and its affiliates
```

Comment:
@cherylwang20 - harmonize with other headers. C to myosuite authors.

Reply:
added in this commit : 89b26e96c685072ccb76467b6d6a2fda67be5094